modules/vaos/app/services/vaos/middleware/vaos_logging.rb

```diff
@@ -72,17 +72,24 @@ def user_session_request?(env)
       # #jti is the value from the JWT key value pair in the response and needed for logging and audit purposes
       #
       # @param env [Faraday::Env] The Request/Response tree object
-      # @param response_env [Faraday::Env::Response] The response part of the tree
       # @return [String] The JTI value or "unknown jti" if a parsing or other error is encountered (failing gracefully)
-      def jti(env, response_env)
+      def jti(env)
         if user_session_request?(env)
-          decode_jwt_no_sig_check(response_env.body)['jti']
+          decode_jwt_no_sig_check(env.body)['jti']
         else
-          decode_jwt_no_sig_check(env.response_headers['X-Vamf-Jwt'])['jti']
+          decode_jwt_no_sig_check(x_vamf_headers(env.request_headers))['jti']
         end
       rescue
         'unknown jti'
       end
+
+      # #x_vamf_headers identifies which X-Vamf-Header was set and returns the appropriate header value
+      #
+      # @param request_headers The set of request headers
+      # @return [String] the JWT set in the request headers
+      def x_vamf_headers(request_headers)
```

Comment:
This is good, but based on some conversation I had I think we'll be able to get rid of this in a future PR. That future PR will involve changing all of our other headers to use the capital version as well. Therefore it will mean basically every single VCR Cassette will need to be manually changed as well. As a result I think it will be best to tackle as a separate tech debt item in the future.

If we have a ticket number for that work we could link to it here.